independent-publisher-connectors/Proposal - 24 Pull Request (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "24 pull request",
+    "description": "connects to '24 pull request' api for open source projects contibution",
+    "version": "1.0",
+    "contact": {
```

Comment:
The contact section should include a "url" : "the product website url"
Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json)

Reply:
fixed